components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```
@@ -7818,6 +7828,94 @@ paths:
             -H "Content-Type: multipart/form-data" -F policySpecFile=@setHeader.yaml -F policyDefinitionFile=@setHeader.j2
             "https://127.0.0.1:9443/api/am/publisher/v3/operation-policies"'
 
+  /operation-policies/export:
+    get:
+      tags:
+        - Import Export
+      summary: |
+        Export common operation policies by their names and versions
```

```suggestion
        Export API policies by their names and versions
```
Since the documentation refer to these as API policies, shall we reword this here in the definition, because ultimately this definition will also go to the doc space?

---

Fixed by [b7f664a](https://github.com/wso2/carbon-apimgt/pull/11325/commits/b7f664af4e393bb1fe5d307dd9dc9c9107e372d6)